pontoon/contributors/templates/contributors/settings.html

```diff
@@ -39,6 +34,18 @@ <h2 id="username">
 
 {% block bottom %}
 <section id="main">
+  <div id="profile-form">
+      <h3>Personal infomation</h3>
+      <div class="field">
+          <label for="profile-username">Username</label>
+          <input type="text" id="profile-username" placeholder="username" value="{{ user.first_name }}" maxlength="30">
```

Comment:
Please [remove](https://www.nngroup.com/articles/form-design-placeholders/) the placeholder attribute.